### PR TITLE
Twitter covid

### PR DESCRIPTION
**Pull request by cjohn281**

### CODE REVIEW

File: twitter-covid/age-spikes-discussion/src/test/scala/EUSpikesTest.scala [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    val filteredDf = filterAgeGroups(spark, df)
+    assert(filteredDf.filter(filteredDf("age_group") === "80+yr").count() == 0)
+  }
+
+  "Dataframe" should "have 2 columns" in {
+    import spark.implicits._
+    val df = pullEUData(spark)
+    val filteredDf = filterAgeGroups(spark, df)
+    val groupedDf = groupData(spark, filteredDf)
+//    println(groupedDf.columns)
```
Comment (TrevorSpear) [inline twitter-covid/age-spikes-discussion/src/test/scala/EUSpikesTest.scala]: delete this

File: twitter-covid/twitter-covid-us/build.sbt [hunk 2]
```diff
@@ -1,19 +1,15 @@
-name := "HashtagByRegion"
+scalaVersion := "2.12.10"
 
-version := "0.1"
+name := "COVID-ANALYSIS-1"
```
Comment (TrevorSpear) [inline twitter-covid/twitter-covid-us/build.sbt]: This should be named HashtagByRegion

File: twitter-covid/twitter-covid-us/build.sbt [hunk 3]
```diff
@@ -1,19 +1,15 @@
...
+organization := "com.revature.michael"
```
Comment (TrevorSpear) [inline twitter-covid/twitter-covid-us/build.sbt]: might just want "com.revature" and have your name in the contributors in the readme

File: twitter-covid/age-spikes-discussion/src/main/scala/Main.scala [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import EUSpikes._
+object Main {
+  def main(args: Array[String]): Unit = {
+    val spark = SparkSession.builder()
+      .appName("EUSpikes")
+      .master("yarn")
```
Comment (TrevorSpear) [inline twitter-covid/age-spikes-discussion/src/main/scala/Main.scala]: might want to comment that to run this locally you should do "*" or .master("local[4]")

File: twitter-covid/period-most-discussed/src/main/scala/Main.scala [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import peakDiscussion._
+object Main {
+    def main(args: Array[String]): Unit = {
+        val spark = SparkSession.builder()
+        .appName("peakDiscussion")
+        .master("yarn") //Switch to "yarn" for running on EMR, "*" for local.
+        .getOrCreate()
+        spark.sparkContext.setLogLevel("WARN")
+        configureAWS(spark) //Only necessary when running locally.
```
Comment (TrevorSpear) [inline twitter-covid/period-most-discussed/src/main/scala/Main.scala:10]: if line 7 is yarn then should this line be commented or does the comment on this line need to change.

**Comment by cjohn281** [comment]
Not ready yet. Almost.

**Pull request closed without merging**